numpy/linalg/linalg.py

```diff
@@ -1142,6 +1144,8 @@ def eig(a):
     _assertNdSquareness(a)
```

Comment:
Mind digging it out? Presumably it required cleaning them up C-side, so this PR is not a subsitute